app/controllers/lists_controller.rb

```diff
@@ -4,6 +4,10 @@ class ListsController < AuthenticatedController
 
   before_action :set_current_board
   before_action :set_list, only: [:show, :edit, :update, :destroy, :move]
+  before_action :set_next_item_and_prev_item, only: :move
+  before_action :set_parent, only: :move
+  # Not at top because we need prev item and next item set first
+  include ValidateMove
```

Comment:
```suggestion
  before_action :set_parent, only: :move
  
  # Not at top because we need prev item and next item set first
  include ValidateMove
```